book/src/tutorial/generate_docs.md

```diff
@@ -47,6 +47,6 @@ Gir will not override them when you re-generate bindings.
 Easy, right.
 If this is your first time publishing a crate, you can find a detailed guide [here](https://doc.rust-lang.org/cargo/reference/publishing.html).
 
-Before you publish the crate, please ensure docs.rs will activate the dox feature and the dox feature of the safe wrapper crate also activates the feature of its dependencies and the unsafe FFI bindings you created.
+Before you publish the crate, please ensure docs.rs will activate the `docsrs` attribute.
```

Comment:
What about when you run cargo doc like we do on the CI side of things?

Reply:
If you have the gtk4-devel package installed, you should be fine without any extra effort.

For annotation of required features with `doc_attr` (e. g. for CI) you need to use nightly as usual and set one env variable for rustdoc, for example: `RUSTDOCFLAGS='--cfg docsrs' cargo +nightly doc`.

Should I mention that in the docs as well?

Reply:
Yes please